server/lib/snippets.js

```diff
@@ -109,7 +109,7 @@ const html = [{
   title: "snippetHTMLExternalStylesheetTitle",
   data: {
     value:
-`<!-- {{ gettext("snippetHTMLExternalStylesheetComment") | safe }} -->
+`<!-- {{ gettext("snippetHTMLExternalStyleSheetComment") | safe }} -->
```

Comment:
I dont understand why this is showing up as a change, is anything different here?